broker-daemon/block-order-worker/index.js

```diff
@@ -101,6 +101,24 @@ class BlockOrderWorker extends EventEmitter {
 
     const blockOrder = new BlockOrder({ id, marketName, side, amount, price, timeInForce })
 
+    // If the user tries to place an order for more than they hold in the counter engine channel, throw an error
+    if (blockOrder.side === BlockOrder.SIDES['BID']) {
```

Comment:
[nit] `BlockOrder.SIDES.BID` would make more sense to me than using a string here

Reply:
@treygriffith @martineehrlich Not sure if this in the scope of this PR, but this is probably a case for setting an `isBid` property on the BlockOrder, so the check would be `if (blockOrder.isBid())`.

Reply:
If we agree on creating the property, we should move the line at 114 too

Reply:
I'm gunna go for it! I think it's a good idea 👍